play/src/front/Phaser/Game/GameManager.ts

```diff
@@ -80,6 +84,37 @@ export class GameManager {
         console.info("Preferred audio input device: " + preferredAudioInputDeviceId);
         console.info("Preferred video input device: " + preferredVideoInputDeviceId);
 
+        // --- MODE INVITÉ ---------------------------------------------
+        if (this.isGuestMode()) {
+            if (!this.playerName) {
+                const random = Math.floor(Math.random() * 10000)
+                    .toString()
+                    .padStart(4, "0");
+                this.playerName = `Invité-${random}`;
```

Comment:
"Invité" is French. I don't think we should translate it using the i18n package (because we want this to be the same for all users, whatever their language is). I think we should add an environment variable: `DEFAULT_GUEST_NAME`, and the default value for this env variable should be "".

If `DEFAULT_GUEST_NAME` is set, we can assume we are in "Guest mode", so you can probably remove the `GUEST_MODE` env var.